### PR TITLE
OPSEXP-2410 Add instruction to install with helm older version of ACS on main helm readme

### PR DESCRIPTION
**Pull request by pmacius**
ref: OPSEXP-2410

### CODE REVIEW

File: docs/helm/README.md [hunk 1]
```diff
@@ -273,6 +273,20 @@ at least 3 nodes with 12 cpu cores and 32 GB of memory in total. You can install
...
 
+To install older versions get the specific values from [this folder](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services).
+Each of files have image tags which override default ones to the specific
+version. Deploying the specific version of ACS can be achieved by executing the
+following:
+
+   ```bash
+   helm install acs alfresco/alfresco-content-services \
+     --values MAJOR.MINOR.N_values.yaml \
+     --values local-dev_values.yaml \
```
Comment (gionn) [inline docs/helm/README.md:284]:
```suggestion
```

this one should be specific to desktop deployment

File: docs/helm/README.md [hunk 2]
```diff
@@ -273,6 +273,20 @@ at least 3 nodes with 12 cpu cores and 32 GB of memory in total. You can install
 requirements by fine tuning the [resource requests](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes)
 available in the [values](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/values.yaml) for each component.
 
+To install older versions get the specific values from [this folder](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services).
```
Comment (gionn) [inline docs/helm/README.md:276]: consider adding an heading so it's more easily searchable and could also be linked from the detailed guides.

just an idea but ideally all the detailed deployment guides we have should at the end link at this document for the general information around helm

File: docs/helm/eks-deployment.md [hunk 3]
```diff
@@ -630,24 +630,7 @@ helm upgrade --install acs alfresco/alfresco-content-services \
...
---set alfresco-repository.persistence.enabled=true \
---set alfresco-repository.persistence.storageClass="nfs-client" \
---set alfresco-transform-service.filestore.persistence.enabled=true \
---set alfresco-transform-service.filestore.persistence.storageClass="nfs-client" \
---set global.known_urls=https://${ACS_HOSTNAME} \
---set global.alfrescoRegistryPullSecrets=quay-registry-secret \
---values letsencrypt_values.yaml \
---namespace=alfresco
-```
+Keep the configuration for helm adding the override file described in this [section](./README.md#previous-versions).
```
Comment (gionn) [inline docs/helm/eks-deployment.md:633]:
```suggestion
Keep the configuration for helm adding the additional override file described in this [section](./README.md#previous-versions).
```

File: docs/helm/desktop-deployment.md [hunk 4]
```diff
@@ -133,23 +133,7 @@ The `helm` command above installs the most current released version of ACS Enter
...
-     --values local-dev_values.yaml \
-     --atomic \
-     --timeout 10m0s \
-     --namespace alfresco
-   ```
-
-> NOTE: The command will wait until the deployment is ready so please be
-patient. See below for
-[troubleshooting](#troubleshooting) tips.
+Keep the configuration for helm adding the override file described in this [section](./README.md#previous-versions).
```
Comment (gionn) [inline docs/helm/desktop-deployment.md:136]:
```suggestion
Keep the configuration for helm adding the additional override file described in this [section](./README.md#previous-versions).
```

File: docs/helm/README.md [hunk 5]
```diff
@@ -285,6 +285,21 @@ There are also several [examples](../helm-examples.md) showing how to deploy wit
 * [Use a custom metadata keystore](https://alfresco.github.io/alfresco-helm-charts/charts/alfresco-repository/docs/keystores.html)
 * [Install ACS license as part of the deployment](https://alfresco.github.io/alfresco-helm-charts/charts/alfresco-repository/docs/enterprise-license.html)
 
+### Previous versions
+
+To install older versions get the specific values from [this folder](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services).
```
Comment (gionn) [inline docs/helm/README.md:290]:
```suggestion
To install older versions of Alfresco, download the relevant values file from [this folder](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services).
```

File: docs/helm/README.md [hunk 6]
```diff
@@ -285,6 +285,21 @@ There are also several [examples](../helm-examples.md) showing how to deploy wit
...
+Each of files have image tags which override default ones to the specific
+version. Deploying the specific version of ACS can be achieved by executing the
+following:
```
Comment (gionn) [inline docs/helm/README.md:293]:
```suggestion
Each file includes image tags that override the default tags, allowing you to deploy a specified ACS version. To deploy a particular ACS version, pass such file as an additional argument to the install command like the following:
```

File: docs/helm/desktop-deployment.md [hunk 7]
```diff
@@ -133,23 +133,7 @@ The `helm` command above installs the most current released version of ACS Enter
...
-     --values local-dev_values.yaml \
-     --atomic \
-     --timeout 10m0s \
-     --namespace alfresco
-   ```
-
-> NOTE: The command will wait until the deployment is ready so please be
-patient. See below for
-[troubleshooting](#troubleshooting) tips.
+Keep the configuration for helm adding the additional override file described in this [section](./README.md#previous-versions).
```
Comment (gionn) [inline docs/helm/desktop-deployment.md:136]:
```suggestion
Use the above helm commands and pass an additional argument as described in this [section](./README.md#previous-versions).
```

File: docs/helm/eks-deployment.md [hunk 8]
```diff
@@ -630,24 +630,7 @@ helm upgrade --install acs alfresco/alfresco-content-services \
...
---set alfresco-repository.persistence.enabled=true \
---set alfresco-repository.persistence.storageClass="nfs-client" \
---set alfresco-transform-service.filestore.persistence.enabled=true \
---set alfresco-transform-service.filestore.persistence.storageClass="nfs-client" \
---set global.known_urls=https://${ACS_HOSTNAME} \
---set global.alfrescoRegistryPullSecrets=quay-registry-secret \
---values letsencrypt_values.yaml \
---namespace=alfresco
-```
+Keep the configuration for helm adding the additional override file described in this [section](./README.md#previous-versions).
```
Comment (gionn) [inline docs/helm/eks-deployment.md:633]:
```suggestion
Use the above helm commands and pass an additional argument as described in this [section](./README.md#previous-versions).
```

**Review by alxgomz — COMMENTED** [review]
I would also add a note that the per version values files are always set with the latest version of each component supported in the "major" version mentioned in the values file name (give an example such as 7.4.N would ship ACS 7.4.2.3 with ADW 4.4.1). We recommend sticking to these but in caase one needs a specific version of a component, he needs to edit the file or further override it using `--set component.image.tag=x.y.z`

**Review by alxgomz — APPROVED** [review]

**Pull request merged**